lib/domgen/ee/templates/abstract_app_server.java.erb

```diff
@@ -89,6 +89,8 @@ public abstract class <%= repository.ee.abstract_app_server_name %>
     glassfish.createCustomResource( "<%= prefix %>/subscriptionsDebugOutputEnabled", false );
     glassfish.createCustomResource( "<%= prefix %>/shouldValidateRepositoryOnLoad", true );
     glassfish.createCustomResource( "<%= prefix %>/requestDebugOutputEnabled", false );
+    glassfish.createCustomResource( "<%= prefix %>/keycloak/username", "keycloak_svc_acct" );
+    glassfish.createCustomResource( "<%= prefix %>/keycloak/password", "keycloak_password" );
```

Comment:
I think this should be guarded by `if repository.keycloak?` and we should add all the required jndi settings. I believe these are all defined at https://github.com/realityforge/domgen/blob/master/lib/domgen/ee/templates/redfish.rb#L117-L129